ee/desktop/menu/dark_mode_darwin.go

```diff
@@ -0,0 +1,18 @@
+//go:build darwin
+// +build darwin
+
+package menu
+
+import (
+	"os/exec"
+)
+
+func isDarkMode() bool {
+	cmd := exec.Command("defaults", "read", "-g", "AppleInterfaceStyle")
```

Comment:
In practice this would only called once as part of building the menu. The menu gets built only when either launcher desktop starts, or control server pushes a new menu/icon.

I borrowed this from some project I found, I'll look that up again. If there is a more formal API we can use, it's going to involve Objective-C.

Reply:
If I think through this, I think we have a bigger issue than it seems.

Users can swap light/dark, so we either need to poll for it, or we need a subscription style API. I don't see discussion about a subscription API, and polling comes with all sorts of downsides.

I do see some suggestions online about using a neutral menu, and then when the menu changes checking the light/dark state. But since I think folks will hang out in the yellow state, this doesn't wholly alleviate the need for polling.

Mostly this feels like a big can of worms.

Reply:
Honestly I think we'd be okay without monitoring light/dark changes, and just build the menu based on whatever the OS says at the time we ask.

Monitoring and reacting to changes is a bunch of complexity to handle what is effectively only a cosmetic bug, which will fix itself the next time launcher restarts. I have zero data to back up this claim, but I feel like users very rarely change their dark/light modes and, at least on Windows, the OS and first party applications themselves don't handle this edge case very well.

Reply:
> Monitoring and reacting to changes is a bunch of complexity

Yes, that's exactly my concern.

On macOS it's common to let the OS swap between them based on time of day :|

Reply:
>On macOS it's common to let the OS swap between them based on time of day :|

oooh, great point. so more frequent than I made it out to be.

Either way, I've removed dark mode stuff from this PR, so we can tackle it separately.

Reply:
Thank you!